banjo/conf/schema.xml

```diff
@@ -219,7 +219,10 @@
    <field name="additional_contributor" type="text_general" indexed="true" stored="true"/>
    <field name="additional_creator" type="text_general" indexed="true" stored="true"/>
    <field name="access_agreement" type="string" indexed="true" stored="true"/>
-
+   <field name="access_agreement_reason_for_change" type="string" indexed="true" stored="true"/>
+   <field name="access_condition_reason_for_change" type="string" indexed="true" stored="true"/>
+   <field name="commercial_status_reason_for_change" type="string" indexed="true" stored="true"/>
+   <field name="access_comments" type="string" indexed="true" stored="true"/>

```

Comment:
@scoen told you so probably because access_agreement contains a pick list of predefined values, so the search has to be exact.
Unless you have a pick list for access_agreement_reason_for_change, it should be text_general.